### PR TITLE
swift: Fix many issues highlighted by codeclimate in proxy.rb

### PR DESCRIPTION
**Pull request by vuntz**

### CODE REVIEW

File: chef/cookbooks/swift/recipes/proxy.rb [hunk 1]
```diff
@@ -107,9 +107,9 @@
 proxy_config[:cross_domain_policy] = cross_domain_policy_l.join("\n")
 
 if node[:platform_family] == "rhel"
-  pkg_list = %w{curl memcached python-dns}
+  pkg_list = %w(curl memcached python-dns)
```
Comment (aplanas) [inline chef/cookbooks/swift/recipes/proxy.rb]: Uhm, I was under the impression that we are moving to backticks for this

Comment (vuntz) [inline chef/cookbooks/swift/recipes/proxy.rb]: Backticks are to replace %x, not %w :-)

Comment (aplanas) [inline chef/cookbooks/swift/recipes/proxy.rb]: I am very sorry for my mistake!
I needed to write this in L253, L266 and so on. But I see that hound was there complaining too.

**Comment by jsuchome** [comment]
+1 from me, but some hound stuff can still be fixed

**Comment by vuntz** [comment]
This was an experiment, closing.

**Pull request closed without merging**